src/main/java/net/minecraftforge/fluids/FluidUtil.java

```diff
@@ -423,21 +425,62 @@ else if (block instanceof BlockLiquid)
     }
 
     /**
+     * Gets the fluid handler for a given fluid's block.
+     * The handler's fill method will place a fluid block with the appropriate amount when not simulated.
+     *
+     * @param world the world the block has to be placed in
+     * @param pos   the position the block has to be placed at
+     * @param fluid the fluid to place
+     * @return the fluid block's fluider handler
+     */
+    private static IFluidHandler getPlacementFluidHandler(final World world, final BlockPos pos, Fluid fluid)
+    {
+        final Block block = fluid.getBlock();
+        IFluidHandler handler;
+        if (block instanceof IFluidBlock)
+        {
+            handler = new FluidBlockWrapper((IFluidBlock) block, world, pos);
+        }
+        else if (block instanceof BlockLiquid)
+        {
+            handler = new BlockLiquidWrapper((BlockLiquid) block, world, pos);
+        }
+        else
+        {
+            handler = new VoidFluidHandler()
```

Comment:
You should probably make a new FluidHandler class instead of an anonymous class here.